administrative-sdk/speechRecording.js

```diff
@@ -322,34 +259,31 @@ class SpeechRecording {
    * @param {Sdk~listSpeechRecordingsCallback} cb The callback that handles the response.
    * @param {Sdk~listSpeechRecordingsErrorCallback} [ecb] The callback that handles the error response.
```

Comment:
Missing documentation update. Callbacks are no longer taken and a promise is returned.